docs/configuration/downstream/koji_build.md

```diff
@@ -76,6 +76,32 @@ Packit dist-git configuration.
    - `all_admins` alias - allowing all users with admin access to the dist-git repo
    - `all_committers` alias - allowing all users with commit access to the dist-git repo
 
+:::info Processing of dist-git events from Pagure
+
+<!-- TODO: Clarify the comment retrigger -->
+```mermaid
+flowchart TD;
+  A[Push event is registered] --> B{"Is build configured
+  for the branch?"}
+  B -. No  .-> NOK((Skip build))
+  B -- Yes --> C{"Has commit been
+  pushed by Pagure?"}
+  C -- Yes --> E{"Is it a PR merge?"}
+  C -. No  .-> D{"Has the commit been pushed
+  by someone in <code>allowed_committers</code>?"}
```

Comment:
These got rendered at least locally for me, so it shouldn't be a problem, just GitHub preview is screwed.